Remove inclusion of libsigc++ headers obsoleted with version 2.6
See also: https://bugs.gentoo.org/show_bug.cgi?id=569700

--- a/examples/console_test/famppct.cpp
+++ b/examples/console_test/famppct.cpp
@@ -30,8 +30,6 @@
 
 #include <sigc++/sigc++.h>
 #include <sigc++/slot.h>
-#include <sigc++/object.h>
-#include <sigc++/object_slot.h>
 
 #include <gtk/gtk.h>
 #include <stdio.h>
--- a/examples/gtk/famppgtk.cpp
+++ b/examples/gtk/famppgtk.cpp
@@ -31,8 +31,6 @@
 
 #include <sigc++/sigc++.h>
 #include <sigc++/slot.h>
-#include <sigc++/object.h>
-#include <sigc++/object_slot.h>
 
 #include <gtk/gtk.h>
 #include <stdio.h>
--- a/src/Fampp2.cpp
+++ b/src/Fampp2.cpp
@@ -34,8 +34,6 @@
 
 #include <sigc++/sigc++.h>
 #include <sigc++/slot.h>
-#include <sigc++/object.h>
-#include <sigc++/object_slot.h>
 
 
 using namespace std;
--- a/src/FamppEvents.hh
+++ b/src/FamppEvents.hh
@@ -36,7 +36,6 @@
 
 #include <fam.h>
 #include <sigc++/signal.h>
-#include <sigc++/object.h>
 #include <SmartPtr.h>
 #include <Singleton.h>
 #include <Factory.h>
